src/vxfw/Border.zig

```diff
@@ -65,6 +75,33 @@ pub fn draw(self: *const Border, ctx: vxfw.DrawContext) Allocator.Error!vxfw.Sur
         surf.writeCell(0, row, .{ .char = .{ .grapheme = "│", .width = 1 }, .style = self.style });
         surf.writeCell(right_edge, row, .{ .char = .{ .grapheme = "│", .width = 1 }, .style = self.style });
     }
+
+    for (self.labels) |label| {
+        const text_len: u16 = @intCast(label.text.len);
```

Comment:
Let's add a guard here

```zig
if (text_len == 0) continue;
```

Reply:
Oh actually we need to change this to be the *actual* width of the line, not just the number of bytes (see another comment about graphemes).

The `DrawContext` object has a method called `stringWidth` that will give you the rendered width of the string.

```zig
   const text_len = ctx.stringWidth(label.text);
   if (text_len == 0) continue;
```